subscription_manager.go

```diff
@@ -38,6 +46,10 @@ func (s *subscriptionManager) deleteAcceptSubscription(id int) {
 	}
 
 	delete(s.acceptSubscriptions, id)
+
+	if len(s.acceptSubscriptions) == 0 {
```

Comment:
This if is probably not even required. The field name still checks out when it is updated with every removal.